commands/deploy.go

```diff
@@ -197,19 +197,34 @@ func RunDeploy(
 			return fmt.Errorf("please provide a --name for your function as it will be deployed on FaaS")
 		}
 
-		envvars, err := parseMap(deployFlags.envvarOpts, "env")
-		if err != nil {
-			return fmt.Errorf("error parsing envvars: %v", err)
+		if err := deployImage(image, fprocess, functionName, deployFlags); err != nil {
```

Comment:
re-factoring this part so that `store deploy` can call directly this code instead of passing by `RunDeploy()`